README.md

```diff
@@ -376,7 +376,7 @@ See the [API documentation](./docs/api.md) for all endpoints.
 
 ### Libraries
 
-- [LangChain](https://python.langchain.com/docs/integrations/llms/ollama) and [LangChain.js](https://js.langchain.com/docs/modules/model_io/models/llms/integrations/ollama) with [example](https://js.langchain.com/docs/use_cases/question_answering/local_retrieval_qa)
+- [LangChain](https://python.langchain.com/docs/integrations/llms/ollama) and [LangChain.js](https://js.langchain.com/docs/integrations/llms/ollama/) with [example](https://js.langchain.com/docs/tutorials/local_rag/)
```

Comment:
Great catch! Thanks. Let's use https://js.langchain.com/docs/integrations/chat/ollama/ for langchain js

Reply:
Sure! Thanks for checking on that. (I was wondering if you'd want to link to llm or chat...)
I updated it!